uk.co.compucorp.civicrm.hrsampledata/tests/phpunit/CRM/HRSampleData/Importer/OptionValueTest.php

```diff
@@ -32,10 +30,15 @@ public function testImport() {
 
     $this->runImporter('CRM_HRSampleData_Importer_OptionValue', $this->rows);
 
-    $this->assertEquals(
-      'Compassionate_Leave',
-      $this->apiGet('OptionValue','name', 'Compassionate_Leave', ['option_group_id' => 'activity_type'])
+    $optionValue = $this->apiGet(
+      'OptionValue',
+      ['option_group_id' => 'activity_type', 'name' => 'Compassionate_Leave']
     );
+
+    foreach($this->rows[0] as $index => $fieldName) {
+      if (!in_array($fieldName, ['option_group_id', 'component_id']))
```

Comment:
always use curly braces, even if the if/else has only one line. Also, the indentation seems wrong here